docs/user_guide/first-event.md

```diff
@@ -0,0 +1,62 @@
+(first-event)=
+
+# Logging your first event!
+
+The `EventLogger` is the main object in Jupyter Events.
+
+```python
+from jupyter_events.logger import EventLogger
+
+logger = EventLogger()
+```
+
+To begin emitting events from a Python application, you need to tell the `EventLogger` what events you'd like to emit. To do this, we should register our event's schema (more on this later) with the logger.
+
+```python
+schema = """
+$id: myapplication.org/example-event
+version: 1
+title: Example Event
+description: An interesting event to collect
+properties:
+   name:
+      title: Name of Event
+      type: string
+"""
+
+
+logger.register_event_schema(schema)
+```
+
+Now that the logger knows about the event, it needs to know _where_ to send it. To do this, we register a logging _Handler_ —borrowed from Python's standard `logging` library—to route the events to the proper place.
```

Comment:
It might be helpful to add a link to the logging docs:
```suggestion
Now that the logger knows about the event, it needs to know _where_ to send it. To do this, we register a logging _Handler_ —borrowed from Python's standard [`logging` library](https://docs.python.org/3/library/logging.html)—to route the events to the proper place.
```

Reply:
Fixed